patterns/footer-with-3-menus.php

```diff
@@ -7,17 +7,17 @@
  */
 ?>
 
-<!-- wp:group {"className":"wc-blocks-footer-pattern","align":"full","style":{"spacing":{"blockGap":"40px","padding":{"top":"var:preset|spacing|30","right":"var:preset|spacing|30","bottom":"var:preset|spacing|30","left":"var:preset|spacing|30"}}}} -->
-<div class="wc-blocks-footer-pattern wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--30);padding-right:var(--wp--preset--spacing--30);padding-bottom:var(--wp--preset--spacing--30);padding-left:var(--wp--preset--spacing--30)">
-	<!-- wp:columns -->
-	<div class="wp-block-columns are-vertically-aligned-top">
-		<!-- wp:column {"verticalAlignment":"top","width":"70%"} -->
-		<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:70%">
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"40px","padding":{"top":"40px","right":"40px","bottom":"40px","left":"40px"}}},"className":"wc-blocks-footer-pattern"} -->
+<div class="wp-block-group alignfull wc-blocks-footer-pattern" style="padding-top:40px;padding-right:40px;padding-bottom:40px;padding-left:40px">
+	<!-- wp:columns {"style":{"spacing":{"padding":{"right":"0","left":"0"}}},"className":"are-vertically-aligned-top"} -->
+	<div class="wp-block-columns are-vertically-aligned-top" style="padding-right:0;padding-left:0">
+		<!-- wp:column {"verticalAlignment":"top","width":"60%"} -->
+		<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:60%">
 			<!-- wp:group {"style":{"spacing":{"blockGap":"32px"}},"layout":{"type":"flex","flexWrap":"wrap","verticalAlignment":"top"}} -->
 			<div class="wp-block-group">
-				<!-- wp:site-logo {"shouldSyncIcon":false} /-->
+				<!-- wp:site-logo {"width":60,"shouldSyncIcon":false} /-->
```

Comment:
I would not recommend hardcoding a default width for the icon on the pattern level; otherwise, we will have a conflict with the Feature within the pattern assembler that exists specifically to configure the icon size:

<img width="1232" alt="Screenshot 2023-10-23 at 14 42 43" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/155df765-138b-4dab-8697-5d2a2e472667">

Reply:
Fixed! :)